### DISCUSSION TITLE
Is the Spotify integration really going away?

### DISCUSSION THREAD

**Discussion by da2x** [Taiko2k/Tauon#728] [Q&A]
I love it. I primarily use it to fill in gaps in my collection. There’s the odd song here and there that’s only available on Spotify.

**Comment by Taiko2k** [comment]
Haven't decided. Main reason I'm moving away from it is lack of reported positive experiences with it. But if you say it's useful to you then that would be an important consideration.

**Reply by da2x** [reply]
The only problem I’ve had that it plays to the wrong playback device. Not sure why, but it turns on my TV (which is off) to play it there instead of using Spotify on the same computer (on). Overall the experience have been great, except for the whole where it’s played part. I enjoy having my Spotify playlists in the same app as my local collection.
